lib/hanami/http/status.rb

```diff
@@ -27,19 +27,24 @@ class Status
       # @example
       #   require 'hanami/http/status'
       #
-      #   Hanami::Http::Status.for_code(418) # => [418, "I'm a teapot"]
+      #   Hanami::Http::Status.for_code(409) # => [409, "Conflict"]
```

Comment:
Actually 418 is probably the only status that is not supported (due to lack of support in `Rack::Utils`)